guides/common/modules/proc_optimizing-performance-by-removing-nics-from-database.adoc

```diff
@@ -0,0 +1,31 @@
+[id="optimizing-performance-by-removing-nics-from-database_{context}"]
+= Optimizing performance by removing NICs from database
+
+Filter and exclude the connections using the `Exclude pattern for facts stored in {Project}` and  `Ignore interfaces with matching identifier` option.
+By default, these options are set to most common hypervisors.
+If you name the virtual interfaces differently, you can update this filter to use it according to your requirements.
+
+.Procedure
+
+. On the {ProjectWebUI}, navigate to *Administer* > *Settings*.
+. Select *Facts*.
+. To filter out all interfaces starting with specific names, for example, `blu`, add `blu*` to the `Ignore interfaces with matching identifier` option.
+. To prevent databases from storing facts related to interfaces starting with specific names, for example, `blu`, add `blu*` to the `Exclude pattern for facts stored in {Project}` option.
++
+By default, it contains the same list as the `Ignore interfaces with matching identifier` option.
+You can override it based on the requirements.
+This filters out facts completely without storing them.
+
+. To remove matching facts from the database (as added in *Administer* > *Settings* > *Facts* > the `Exclude pattern for facts stored in {Project}` option), enter the following command:
```

Comment:
It's unclear to me _when_ and _for what/why_ to run this command. Do I have to run the command for the changes made in settings to come into effect?

Reply:
To optimize the performance you can remove facts and interfaces from the database. to remove interfaces and facts those start with specific names you can set preferences (step 2 and 3) and to remove the facts and interfaces matching with those interfaces, you can run these commands. @lzap please add if I am missing something. :)

Reply:
Sounds much better in my opinion.

Reply:
@maximiliankolb, so should I add this information in the procedure intro? This is there at the end of the concept module in different words.

Reply:
Whatever you see fit best to comply with https://github.com/theforeman/foreman-documentation/tree/master/guides#structure.

Right now, I don't like the half-sentence in the brackets and would like you to reword this in some shape or form.

Maybe something like:
```
.Removing facts from ...
[source....]
----
# some command
---

.Removing nics from ...
[source....]
----
# some other command
---
```

this would not result in a list, but small titles plus code blocks.

Reply:
Removed the part in the parenthesis () from steps and added it as a step result immediately after the step. Hopefully this should resolve the issue. Kindly check the latest commit and confirm. Thank you :)